teams/index.html

```diff
@@ -403,4 +386,84 @@ <h4 class="text-center">Wilma Woudenberg</h4>
         <hr class="whitespace">
         <!-- end  Development teams -->
         
+         <!-- begin Former team members -->
+            <hr class="whitespace">
+            <div class="row text-center">
+                <div class="small-12 columns">
+                    <h3>Former members of the OME team in Dundee</h3>
```

Comment:
"in Dundee": so might Harald someday move down here?

Reply:
Well, strictly it's paid by Uni of Dundee since Seb isn't based here physically either, but I'm not sure we need to be that nitpicky for this context!

Reply:
Okay. :smiley: